Rick-and-Morty/Rick And Morty/Views/CharacterCard.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  CharacterTileView.swift
+//  Rick And Morty
+//
+//  Created by Scottie Gray on 2021-08-10.
+//  Copyright © 2021 Novoda. All rights reserved.
+//
+
+import SwiftUI
+import Foundation
+
+struct CharacterCardState {
+    var id: Int
+    var name: String
+    var image: UIImage
+    var isAlive: Bool
+    var species: String
+    var lastLocation: String
+    var firstEpisode: String
+}
+
+struct CharacterCard: View {
+    var characterCardState: CharacterCardState
+    
+    private let cornerRadius: CGFloat = 10
+    private let noSpacing: CGFloat = 0
+    private let VSpacing: CGFloat = 10
```

Comment:
Id probably extract these into their own private constants struct. Something we do on Glovo which i quite like, except they use an enum which i dont like. 

```
private struct Constants {
    let cornerRadius: CGFloat = 10
    let noSpacing: CGFloat = 0
    let VSpacing: CGFloat = 10
}
```
then id add the other magic numbers dotted around to it too like on line 43, 67 &70